views/objecting-entity-name.html

```diff
@@ -25,13 +25,10 @@
       <form method="post">
         {{ govukInput({
           label: {
-            text: "What is your full name or the name of your organisation?",
+            text: textFullName,
             isPageHeading: true,
             classes: "govuk-label--xl"
```

Comment:
Would you be able to change the class to be `govuk-label--l` instead please? This will then match the prototype

Reply:
Thanks Tori! Fix done